socketio.js

```diff
@@ -180,23 +179,24 @@ io.on("connection", function (socket) {
 		);
 	});
 
-	socket.on("open_in_editor", (data) => {
+	socket.on("open_in_editor", async (data) => {
 		let s = get_redis_subscriber("redis_queue");
-		s.publish("open_in_editor", JSON.stringify(data));
+		await s.publish("open_in_editor", JSON.stringify(data));
 	});
 });
```

Comment:
Any reason we need to async-await this? Doesn't seem to be required to me. :eyes:

Reply:
I'm not a js crack, but `get_redis_subscriber` is now async as well as `publish` which makes the entire function async.